WindowsServerDocs/storage/refs/refs-overview.md

```diff
@@ -59,27 +59,48 @@ ReFS is designed to support extremely large data sets--millions of terabytes--wi
 
 ## Supported deployments
 
+Microsoft has developed NTFS specifically for general-purpose use with a wide range of configurations and workloads, however for customers specially requiring the availability, resiliency, and/or scale which ReFS provides, Microsoft supports ReFS for use under the following configurations and scenarios. 
+
+>[!NOTE]
+> All ReFS supported configurations must use Windows HLK certified hardware and meet application requirements. 
```

Comment:
I think instead of Windows HLK, we should say [Windows Server Catalog](https://www.windowsservercatalog.com/)

Reply:
Agreed. That is more clear.